response_operations_ui/templates/conversation-view/cv-message-block.html

```diff
@@ -0,0 +1,22 @@
+{% for message in messages %}
+<div class="secure-message-sent-message">
+    {% if message.get('internal') == True %}
+        <div name="sm-from-ons" id="sm-from-ons-{{ loop.index }}" class="secure-message-sent-message-meta sm-from-ons">
+    {% else %}
+        <div name="sm-from-respondent" id="sm-from-respondent-{{ loop.index }}" class="secure-message-sent-message-meta sm-from-respondent">
+    {% endif %}
+    {% if loop.last %}<div name="latest-message" id="latest-message"></div>{% endif %}
+        <span class="secure-message-sent-message-meta-from venus">
+            From:
+            <span name="sm-from-user" id="sm-from-user-{{ loop.index }}" style="display:inline;">{{ message.get('username') }}</span>
```

Comment:
inline style?

Reply:
Yeah, unfortunately there currently isn't a `display: inline` utility class - have raised a ticket to add one to the pattern library.